src/Razor/Microsoft.AspNetCore.Mvc.Razor.Extensions/test/IntegrationTests/CodeGenerationIntegrationTest.cs

```diff
@@ -19,7 +19,7 @@ public CodeGenerationIntegrationTest()
             : base(generateBaselines: null, projectDirectoryHint: "Microsoft.AspNetCore.Mvc.Razor.Extensions")
         {
             Configuration = RazorConfiguration.Create(
-                RazorLanguageVersion.Version_3_0,
+                RazorLanguageVersion.Latest,
```

Comment:
This fixes some tests that otherwise fail because the baselines would need to change. I don't see any clear reason for this particular set of unit tests to be hardcoded to razor language version 3, and it was only passing before because these tests happened not to exercise any behaviors that have changed between versions 3 and 5.

I could have fixed the broken tests by changing the baselines instead, but I think we actually want the test coverage to be for razor language version 5 as much as possible, except for the small number of tests that are explicitly about covering legacy behaviors.